packages/components/Checkbox/src/Checkbox.tsx

```diff
@@ -54,7 +54,7 @@ export const Checkbox = compose<CheckboxType>({
       return (
         <Slots.root {...mergedProps} {...(Platform.OS == 'android' && { accessible: !disabled, focusable: !disabled })}>
           {Checkbox.state.labelIsBefore && labelComponent}
-          <Slots.checkbox {...(Platform.OS == 'android' && { onPress, disabled })} accessible={false} focusable={false}>
+          <Slots.checkbox accessible={false} onPress={onPress} disabled focusable={false}>
```

Comment:
Won't this make it always disabled, and therefore onPress won't fire?

Reply:
Nevermind! This is fine =)

Reply:
hmm correct as we extracted `disabled`, so it takes either undefined or true, hence makes it work.